pvlib/modelchain.py

```diff
@@ -260,6 +263,27 @@ def get_orientation(strategy, **kwargs):
     return surface_tilt, surface_azimuth
 
 
+@dataclass
+class ModelChainResult:
+    T = TypeVar('T')
+    PerArray = Union[T, Tuple[T, ...]]
+    # system-level information
+    solar_position: Optional[pd.DataFrame] = field(default=None)
+    airmass: Optional[pd.DataFrame] = field(default=None)
+    ac: Optional[pd.Series] = field(default=None)
+    # per DC array information
+    tracking: Optional[pd.DataFrame] = field(default=None)
```

Comment:
Yes, because we don't have support for multiple arrays with trackers we need to keep treating is as a system-wide quantity. In the future support will (should) be added for multiple arrays each with its own tracker. At that time this would need to be changed to a `PerArray`.